plugins/modules/postgresql_pg_hba.py

```diff
@@ -90,13 +90,35 @@
     type: str
     default: sdu
     choices: [ sdu, sud, dsu, dus, usd, uds ]
+  overwrite:
+    description:
+      - Remove all existing rules before adding rules. (Like I(state=absent) for all pre-existing rules.)
+    type: bool
+    default: false
   keep_comments_at_rules:
     description:
       - If C(true), comments that stand together with a rule in one line are kept behind that line.
       - If C(false), such comments are moved to the beginning of the file, like all other comments.
     type: bool
     default: false
     version_added: '1.5.0'
+  rules:
+    description:
+      - A list of objects, specifying rules for the pg_hba.conf. Use this to manage multiple rules at once.
+      - "Each object can have the following keys (the 'rule-specific arguments'), which are treated the same as if they were arguments of this module:"
+      - C(address), C(comment), C(contype), C(databases), C(method), C(netmask), C(options), C(state), C(users)
+      - See also C(rules_behavior).
+    type: list
+    elements: dict
+  rules_behavior:
+    description:
+      - "Configure how the I(rules) argument works together with the rule-specific arguments outside the I(rules) argument."
+      - See I(rules) for the complete list of rule-specific arguments.
+      - When set to C(conflict), fail if I(rules) and, for example, I(address) are set.
+      - If C(combine), the normal rule-specific arguments are not defining a rule, but are used as defaults for the arguments in the I(rules) argument.
```

Comment:
```suggestion
      - If C(combine), the normal rule-specific arguments are not defining a rule, but are used as defaults for the arguments in the I(rules) argument.
      - Is used only when I(rules) is specified, ignored otherwise.
```